app/models/submission-internal-review.js

```diff
@@ -0,0 +1,12 @@
+import { belongsTo, hasMany, attr } from '@ember-data/model';
+import ModelWithModifier from 'frontend-kaleidos/models/model-with-modifier';
+
+export default class SubmissionInternalReviewModel extends ModelWithModifier {
```

Comment:
I need to test the concurrency still. that throws an error which will be uncaught and break logic.
Do we really need the `ModelWithModifier`?
We could try without. Worst case they lose a few lines of text? 
Or we need a way where we don't error but just warn?
I'm inclined to take it out because all the points of error need to be surrounded with try catch

Reply:
we could also reload `internalReview.submissions` before save since losing connection to a BIS submission is likely to happen when editing a design agenda when BIS update is ongoing?

Reply:
I think we are keeping the concurrency check for now.
Can be finetuned later if we were to extract this bit from the description panels and save it separately.
we did have to do a `internalReview.hasMany('submissions').reload();` for concurrency issues